packages/story-editor/src/components/toggleButton/index.js

```diff
@@ -76,7 +102,7 @@ export const ToggleButton = forwardRef(
     return (
       <Tooltip
         hasTail
-        title={label}
+        title={!hideTooltip && label}
```

Comment:
Looks like marcin added it in #10847, but I think it makes sense that there is no tooltip here. The toggle button has text in it, and adding a tooltip imo wouldn't add any more information that it doesn't already have.

![image](https://user-images.githubusercontent.com/22185279/158613786-c4d4963a-dd3c-4671-9b45-b2cda2bf393f.png)

I think the tooltip would be necessary if there was no text in the button

Reply:
Hm, okay, but the `ToggleButton` for Layers doesn't actually have a `label` prop passed (as far as I can see), so it shouldn't display the tooltip anyway, right? Just wondering if it needs all the `hideTooltip` logic at all.

Reply:
From testing: yep, looks like the tooltip doesn't show anyway since `label` is not assigned. So perhaps the `hideTooltip` part can just be removed? (cc @BrittanyIRL since from the commits it looks like it was added after taking over from Marcin)

Reply:
Ah I see - just remove that. Nice catch!

Reply:
i'd removed it because it was redundant - it says the same thing the button does. but it's fine either way with me.